app/controllers/articles_controller.rb

```diff
@@ -132,7 +132,8 @@ def delete_confirm
   def destroy
     authorize @article
     @article.destroy!
-    Notification.remove_all(id: @article.id, class_name: "Article", action: "Published")
+    Notification.remove_all_without_delay(id: @article.id, class_name: "Article", action: "Published")
```

Comment:
Hmmmm seems like this would timeout because there could be thousands of these.

_Unless_ the implementation of `remove_all` called `delete_all` instead of `destroy_all` because then it would do it in bulk. I feel like this would probably be fine to do.